fusor/model.py

```diff
@@ -1,11 +1,10 @@
 """Model for fusion class"""
 import json
-from pydantic import BaseModel, validator, StrictInt, StrictBool, StrictStr, \
-    root_validator
+from pydantic import BaseModel, validator, StrictInt, StrictBool, StrictStr
 from typing import Optional, List, Union
-from gene.schemas import GeneDescriptor, SequenceLocation, ChromosomeLocation
 from enum import Enum
-from variation.schemas.ga4gh_vrs import SequenceState
+from ga4gh.vrsatile.pydantic.vrsatile_model import GeneDescriptor, \
+    LocationDescriptor, SequenceDescriptor
 
 
 def check_curie(cls, v):
```

Comment:
We should remove this and do something like this where fields are supposed to be CURIEs.
```
from ga4gh.vrsatile.pydantic.vrs_model import CURIE

class Example(BaseModel):
    field: CURIE
```
We have the regex pattern from the VRS schema defined [here](https://github.com/ga4gh/vrs/blob/main/schema/vrs.yaml#L648). I think I should make an issue, if it doesn't already exist, to update this since it allows some invalid CURIES (`i.e: `e:     `)